web/src/server_events_dispatch.js

```diff
@@ -233,7 +233,6 @@ export function dispatch_normal_event(event) {
                 move_messages_between_streams_limit_seconds: noop,
                 move_messages_within_stream_limit_seconds: message_edit.update_inline_topic_edit_ui,
                 message_retention_days: noop,
-                move_messages_between_streams_policy: noop,
```

Comment:
I guess you would need to add the new setting here for live-update to work correctly.